Web proto endpoints

omo7765 opened the pull request.

omo7765 committed:
trying to figure out how to get data from server to client

omo7765 committed:
trying to figure out how to get data from server to client

omo7765 committed:
trying to figure out how to get data from server to client

omo7765 committed:
fixed an issue with the enter-room endpoint

omo7765 committed:
fixed an issue with the enter-room endpoint

omo7765 committed:
fixed an issue with the enter-room endpoint

omo7765 committed:
fixed an issue with the enter-room endpoint

omo7765 committed:
used storage utility to keep track of names. need to update participants list

omo7765 committed:
used storage utility to keep track of names. need to update participants list

omo7765 committed:
Merge branch 'web-proto-endpoints' of https://github.com/SI-Squad/stellae-bellum into web-proto-endpoints

omo7765 committed:
idk

omo7765 committed:
update

omo7765 committed:
updated some linking between home page and create pages.
Also got list ot display.

omo7765 committed:
Merge branch 'web-proto-endpoints' of https://github.com/SI-Squad/stellae-bellum into web-proto-endpoints

omo7765 committed:
updated the readme, got the list to display and be on new lines

omo7765 committed:
finished update list and format

omo7765 committed:
Merge branch 'master' into web-proto-endpoints

yxd3549 commented on application/server.py:
```diff
@@ -66,6 +66,13 @@ def handle_create_room_form():
         confirmed_password = request.form.get('confirmed-room-password')
 
         if room_password == confirmed_password:
+<<<<<<< HEAD
```
Ummm. I think this is the result of a merge conflict. You'll need to solve this.

omo7765 replied:
i believe i fixed it now

yxd3549 reviewed (changes requested):
Good stuff! Just address the one comment regarding the merge conflict

omo7765 committed:
fixed merge confilct

omo7765 committed:
linked gameover and gameroom to home

omo7765 committed:
connect gameroom to gameover via button

omo7765 committed:
connects waiting room and open room to build (create board)
connect build to gameroom
(when you click done, it will take you to gameroom as well as getboard())

omo7765 commented:
also did all of the connecting of web prototype. created a button for some pages that may need to be changed for game logic

yxd3549 reviewed (approved):
Great work!!
